taker-frontend/src/components/Tour.tsx

```diff
@@ -0,0 +1,182 @@
+                        <br />
+                        <br />
+                        Additionally, CFD trading is inherently risky, make sure to read up so you don't get rekt.
+                        <br />
+                        <br />
+                        Follow the tour for a basic intro, for more questions check the&nbsp;<FaqLink />.
+                    </Text>
+                </>
+            ),
+            position: "right",
+        },
+        {
+            element: "#walletSwitchButton",
+            intro:
+                "Before you can open a position you will have to add funds to your wallet. We will come back to this at the end of the tour.",
+            position: "right",
+        },
+        {
+            title: "Long and Short",
+            intro: (
+                <>
+                    <Text>
+                        When opening a long position you make profits if the Bitcoin price goes up.
+                        <br />
+                        When opening a short position you make profits if the Bitcoin price goes down.
+                        <br />
+                        <img src={cfd101} width={"400px"} alt={"CFD 101"} />
+                    </Text>
+                </>
+            ),
+            element: "#longShortButtonSwitch",
+            position: "right",
+        },
+        {
+            element: "#makerLongPrice",
+            intro: "This is the current price of the maker for opening a long position...",
+            position: "right",
+        },
+        {
+            element: "#longQuantityInput",
+            intro:
+                "You specify how many contracts of BTC/USD you buy. This will determine the margin that will be locked up on chain.",
+            position: "right",
+        },
+        {
+            element: "#longLeverage",
+            // TODO: Add link to learn about leverage trading
+            intro: "The leverage influences the margin as well. At the moment the leverage is fixed at x2.",
+            position: "right",
+        },
+        {
+            element: "#longRequiredMargin",
+            intro:
+                "This is the amount of BTC that is necessary to open the position. This amount will be locked on chain.",
+            position: "right",
+        },
+        {
+            element: "#longPerpetualCost",
+            intro:
+                "To allow you to close any point in time in the future you pay a small fee per the hour. Initially you will pay fees for 24h, with every hour that the CFD remains open you pay for one additional hour.",
+            position: "right",
+        },
+        {
+            element: "#longButton",
+            intro: (
+                <>
+                    <Text>
+                        Clicking this button will open a confirmation dialog similar to this:
+                        <br />
+                        <img src={confirmationDialog} width={"400px"} alt={"Confirm it already!"} />
```

Comment:
I would remove this image. It will be outdated too fast, e.g. this PR #1960 makes changes to the UI already.

Reply:
You could probably include the dialog component and fill it with dummy values.

Reply:
> You could probably include the dialog component and fill it with dummy values.

This *might* be hard to achieve given the render lifecycle and the tour lifecycle. I would not try to achieve this, it will be more headache than it helps.
I think the screenshot is fine - even if it gets outdated. It helps the user understand that "something" will pop up, I don't think users will try to understand it. The point is that the *know* that they can press the `Long` button and it won't open the position immediately.

Reply:
you already mention in the text that a dialog will be opened, if the image is outdated, then it does not add much value. Imho anyone can imagine how a confirmation dialog will look like :D .

Reply:
> you already mention in the text that a dialog will be opened, if the image is outdated, then it does not add much value. Imho anyone can imagine how a confirmation dialog will look like :D .

I think the image helps drawing attention to it. Text is easily ignored, an image will draw your attention.
If it's a big deal I'd remove it, I still thinks it adds more value than it brings problems 🤷‍♂️